modules/worker-utils/src/lib/process-utils/child-process-proxy.ts

```diff
@@ -39,7 +40,7 @@ const DEFAULT_PROPS: ChildProcessProxyProps = {
 export default class ChildProcessProxy {
   id: string;
   props: ChildProcessProxyProps = {...DEFAULT_PROPS};
-  private childProcess: ChildProcess | null = null;
+  private childProcess: ReturnType<typeof ChildProcess['spawn']> | null = null;
```

Comment:
If `import type` works, then we can drop this. If it does not work, can we define `type ChildProcess = ...` at the top of the file, with a comment explaining why we are doing this?